### PR TITLE
Feat/custom change addr

### PR DESCRIPTION
**Pull request by isocolsky**

### CODE REVIEW

File: lib/verifier.js [hunk 1]
```diff
@@ -86,14 +86,21 @@ Verifier.checkProposalCreation = function(args, txp) {
 
   if (txp.outputs.length != args.outputs.length) return false;
 
-  _.each(txp.outputs, function(o1, i) {
+  for (var i = 0; i < txp.outputs.length; i++) {
```
Comment (matiu) [inline lib/verifier.js]: Much better!

**Comment by matiu** [comment]
ACK!

**Comment by isocolsky** [comment]
Thank you for the review!

File: lib/verifier.js (hunk 1 above)
Comment (isocolsky) [inline lib/verifier.js]: Back to the roots ;)

**Pull request merged**